### PR TITLE
Styling

### LINKED ISSUE #4: Styling
Try to match ETDA Workflow. Blacklight's color scheme is already similar. The logo + header should match ETDA Workflow.

### PR DESCRIPTION
**Pull request by ajkiessl**
Basic styling to match ETDA Workflow, and look like a PSU Libraries site.  I took the path of least resistance and changed as little as possible.  I'm open to suggestions to make it look better if we think that's needed.

Screen Shot 2022-06-14 at 3 09 36 PM

I need to update etda_utilities before this gets merged.

closes #4

### CODE REVIEW

**Review by whereismyjetpack — CHANGES_REQUESTED** [review]

File: app/models/workflow_url.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    end
+
+    private
+
+      def partner
+        current_partner.id
+      end
+
+      def host
+        ENV['WORKFLOW_HOST'] || 'prod'
```
[the remark this answers is not in the document]
Comment (ajkiessl) [reply app/models/workflow_url.rb:17]: Oh right, I'll fix this.

[the remark this answers is not in the document]
Comment (ajkiessl) [reply app/models/workflow_url.rb]: Might as well try to refactor this into utilities tomorrow.

**Review by banukutlu — APPROVED** [review]
this looks good to me 👍🏻 once the lint issues get fixed and with the WORKFLOW_HOST refactor, it is a good start, we can always add on as needed. thanks @ajkiessl!

**Comment by ajkiessl** [comment]
This also closes #26 .  I added the 'Add My Work' link since it uses the WORKFLOW_HOST.  While I was doing that, I decided just to finish up #26 and add the about page.  I don't think we need a 'Home' link since we don't really have a home page.  

I'm going to get https://github.com/psu-stewardship/etda_utilities/pull/20 merged and update etda_utilities.  This should be ready to go after that.

**Review by whereismyjetpack — APPROVED** [review]

**Pull request merged**